### PR TITLE
lenght > length

### PR DESCRIPTION
**Pull request by brianteeman**
This changes code so please check carefully

### CODE REVIEW

**Review by Elfangor93 — APPROVED** [review]
Looks good to me.

File: administrator/com_joomgallery/src/Service/Migration/Migration.php [hunk 1]
```diff
@@ -1741,21 +1741,21 @@ protected function applyConvertData(array $data, array $mapping, string $pk_name
           // Get source lenght
```
Comment (MrMusic) [inline administrator/com_joomgallery/src/Service/Migration/Migration.php:1741]: Last typo?

Comment (brianteeman) [reply administrator/com_joomgallery/src/Service/Migration/Migration.php:1741]: that should have been fixed in the bigger typo pr

**Comment by MrMusic** [comment]
I have tested this PR ✅ successfully.
Thank you very much!

**Pull request merged**